website/blog/2023-1-4-backups-on-your-coffee-break.md

```diff
@@ -0,0 +1,81 @@
+Now your cell phone’s ringing off the hook.
+Slapping your forehead with the palm of your hand, you shout,
+“If only backups were faster and easier!”
+
+<!-- truncate -->
+
+Regular backups are increasingly important and must be prioritized; even over your coffee break. A recent study by
+[Arlington Research](https://www.businesswire.com/news/home/20210511005132/en/An-Alarming-85-of-Organizations-Using-Microsoft-365-Have-Suffered-Email-Data-Breaches-Research-by-Egress-Reveals#:~:text=15%25%20of%20organizations%20using%20Microsoft,data%20in%20error%20via%20email.)
+found that 85% of organizations using Microsoft 365 suffered email data breaches in the six months prior to May 2021.
+And it’s not just malware that threatens to corrupt data; downtime can have equally devastating impacts.
+[Two out of every five servers](https://www.veeam.com/blog/data-loss-2022.html)
+experienced an outage over the past 12 months.
+Data can also be lost or corrupted during poorly executed migrations or the cancellation of a software license
+or by human error. And once it’s gone, it’s gone, unless you’ve backed it up.
+
+The question really is: why wouldn’t you back up your M365 data?
+
+IDC estimates that [six out of every 10 organizations](https://www.dsm.net/idc-why-backup-for-office-365-is-essential)
+don’t have a data protection plan for their Microsoft 365 data.
+Why? Because, historically, M365 backups have been slow,
+tedious and expensive, requiring complex workflows and scripts, and constant supervision:
+
+- Companies often face physical limitations of their storage devices, such as servers, external hard drives, or other media.
+They may have to choose what data to backup or compromise on how often they back it up.
+- Backups can be time-consuming, especially without automation.
+Often, someone has to monitor the process to address any issues that arise. With their to-do list growing day by day,
+IT security teams must often prioritize more urgent work.
+- Manual backups aren’t just slow and tedious, they’re unreliable. When work is busy, or when your employee’s stomach
+is growling -it may be pushed to the bottom of the priority list.
+
+Considering these challenges, it’s clear to see why an IT security staffer might put backups on the back burner.
+
+A Faster, Easier Way to Back up Your Data
+
+Fortunately, [Corso](https://corsobackup.io/), a free and open-source tool, is enabling IT administrators to backup all
+their M365 data during their morning coffee break -or while their lunch is in the microwave. Here’s how:
+
+- Purpose-built for Microsoft 365, Corso provides comprehensive backup and restore workflows that slash backup time and overhead.
+- It’s free: because Corso is 100% open-source. Flexible retention policies reduce storage costs, as well. Corso works
+with any S3-compatible object storage system, including AWS, Google Cloud, Backblaze and Azure Blob.
+- It’s fast! Corso doesn’t use unreliable scripts or workarounds. Instead,
```

Comment:
Maybe mention that it is written in Go, a modern programming language that came out of Google that has been purpose-built for systems programming.